lib/n_.js

```diff
@@ -18,9 +18,11 @@ var repl = require('repl'),
 // create new pristine `lodash` instance
 var _ = require('lodash').runInContext(server.context);
 
+var lodashToInject = process.argv.indexOf('--fp') === -1 ? _ : require('lodash/fp').runInContext();
```

Comment:
Ok, so it's to avoid having mixins in the REPL that may be used in the code in this project, or any code using the project programmatically... Thanks